stmt/stmts/account_manage_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2015 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stmts_test
+
+import (
+	. "github.com/pingcap/check"
+)
+
+func (s *testStmtSuite) TestCreateUserStmt(c *C) {
+	// Make sure user test not in mysql.User.
+	tx := mustBegin(c, s.testDB)
+	rows, err := tx.Query(`SELECT Password FROM mysql.User WHERE User="test" and Host="localhost"`)
+	c.Assert(err, IsNil)
+	rows.Next()
+	c.Assert(rows.Next(), IsFalse)
```

Comment:
No, those lines are useful. I use them to make sure there is only one row for certain condition.